.vscode/launch.json

```diff
@@ -31,16 +31,16 @@
       "request": "launch",
       "console": "internalConsole",
       "preLaunchTask": "build",
-      "program": "${workspaceRoot}/tools/import/bin/Debug/net5.0/Pims.Tools.Import.dll",
```

Comment:
This whole section can be safely removed - we don't have the data import tool in the solution anymore